pkg/mqtt5/config/type_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2023 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/validator"
+)
+
+func TestValidate(t *testing.T) {
+	validConfig := Mqtt5Config{
+		Host:       "localhost",
```

Comment:
add an unit test for host is an ipv4 address and ipv6 address, such as `127.0.0.1` and `::1`

Reply:
added